Makefile

```diff
@@ -1,14 +0,0 @@
-# set env value NAME
-NAME = "raid2earn"
-# set env value VERSION
-VERSION = "1.0.0"
-
-build:
-	go build -o bin/$(NAME) -ldflags "-X main.Version=$(VERSION)" $(NAME).go
-
-run:
-	make build
-	./bin/$(NAME)
-
-start:
-	./bin/$(NAME)
```

Comment:
As in the previous PR comment. Please let's restore the `Makefile`

Reply:
Okay. Can I replace "raidtoearn" in Makefile to "Golang server template"

Reply:
replace it with `app`

Reply:
okay